server/polar/webhook/webhooks.py

```diff
@@ -250,6 +257,84 @@ def get_slack_payload(self, target: User | Organization) -> str:
         return json.dumps(payload)
 
 
+class WebhookOrderRefundedPayload(BaseWebhookPayload):
+    """
+    Sent when an order is fully or partially refunded.
+
+    **Discord & Slack support:** Full
+    """
+
+    type: Literal[WebhookEventType.order_refunded]
+    data: OrderSchema
+
+    def get_discord_payload(self, target: User | Organization) -> str:
+        if isinstance(target, User):
+            raise UnsupportedTarget(target, self.__class__, WebhookFormat.discord)
+
+        amount_display = self.data.get_refunded_amount_display()
+
+        fields: list[DiscordEmbedField] = [
+            {"name": "Product", "value": self.data.product.name},
+            {"name": "Refunded", "value": amount_display},
+            {"name": "Customer", "value": self.data.customer.email},
+        ]
+        if self.data.subscription is not None:
+            fields.append({"name": "Subscription", "value": "Yes"})
```

Comment:
Is it really useful? 😊 They can probably guess it from the product name.

Reply:
Fair point. Leaving for now though to mimic `OrderCreated` since it's weird if they're different for the same object in that sense, but might remove on both.